Warcraft.NET/Files/ADT/Chunks/Legion/MLHD.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.IO;
+using Warcraft.NET.Extensions;
+using Warcraft.NET.Files.Interfaces;
+using Warcraft.NET.Files.Structures;
+
+namespace Warcraft.NET.Files.ADT.Chunks.Legion
+{
+    /// <summary>
+    /// MLHD Chunk - Level of detail header
+    /// </summary>
+    public class MLHD : IIFFChunk, IBinarySerializable
+    {
+        /// <summary>
+        /// Holds the binary chunk signature.
+        /// </summary>
+        public const string Signature = "MLHD";
+
+        /// <summary>
+        /// Unknown uint
+        /// </summary>
+        public uint Unknown0 { get; set; }
+
+        /// <summary>
+        /// Unknown bounding box
+        /// </summary>
+        public BoundingBox UnkBoundingBox { get; set; }
+
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="MLHD"/> class.
+        /// </summary>
+        public MLHD()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="MLHD"/> class.
+        /// </summary>
+        /// <param name="inData">ExtendedData.</param>
+        public MLHD(byte[] inData)
+        {
+            LoadBinaryData(inData);
+        }
+
+        /// <inheritdoc/>
+        public void LoadBinaryData(byte[] inData)
+        {
+            using (var ms = new MemoryStream(inData))
+            using (var br = new BinaryReader(ms))
+            {
+                Unknown0 = br.ReadUInt32();
+                UnkBoundingBox = new BoundingBox(br.ReadVector3(AxisConfiguration.Native), br.ReadVector3(AxisConfiguration.Native));
```

Comment:
Maybe remove the Structures.AxisConfiguration.Native and use ReadBoundingBox/WriteBoundingBox